File: Extras x RetosJS/7 | Acronimo/7-Extra.js
/*
    * Genera un acrónimo a partir de una frase, usando las primeras letras de cada palabra.
*/

// ? Solución

const generarAcronimo = (frase) => {
	let palabrasFrase = [];
	palabrasFrase = frase.toUpperCase().split(' ');
	cantidadPalabras = palabrasFrase.length;
	let acronimo = [];
	for (let i = 0; i < cantidadPalabras; i++) {
		acronimo.push(palabrasFrase[i].charAt(0));
	}
	console.log(acronimo.join(''));
};

// Todo: Pruebas
generarAcronimo('Prueba contingente de salud') // Resultado: PCDS

File: Extras x RetosJS/7 | Acronimo/7-Extra.py
# Genera un acrónimo a partir de una frase, usando las primeras letras de cada palabra.

def generateAcronym(phrase):
    wordsPhrase = phrase.upper().split()
    countWords = len(wordsPhrase)
    acronym = []

    for i in range(countWords):
        acronym.append(wordsPhrase[i][0])

    print(''.join(acronym))

generateAcronym('Prueba contingente de salud')